neo/protocol.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
We can made this optional without remove the current file

Reply:
Currently, NuGet doesn't support pack with this file. I will not remove it from neo-cli. But there is no reason to keep it in this repo.